### PR TITLE
Extended PI for conv

### PR DESCRIPTION
**Pull request by pacospace**
Extended PI:
- conv is renamed conv2d now.
- added conv1d.
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: tensorflow/conv1d.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        tensor_input_width=_ARGS_INPUT_WIDTH,
+        tensor_input_channels=_ARGS_T_INPUT_CHANNELS,
+        filter_width=_ARGS_FILTER_WIDTH,
+        filter_input_channels=_ARGS_F_INPUT_CHANNELS,
+        filter_output_channels=_ARGS_OUTPUT_CHANNELS
+    )
+
+    result = {
+        "framework": "tensorflow",
+        "name": "conv1D",
```
Comment (fridex) [inline tensorflow/conv1d.py]: `conv1d`?

**Review by fridex — COMMENTED** [review]
Nice! :+1: :medal_military: One comment left.

File: tensorflow/conv1d.py (hunk 1 above)
Comment (fridex) [reply tensorflow/conv1d.py]: also in conv2d.py -> `conv2d`?

Comment (pacospace) [reply tensorflow/conv1d.py]: Yes, for natural language processing applications for example, you can use also convolution. In that case you require conv1d to extract features from a sentence, which is 1d typically.

Comment (fridex) [reply tensorflow/conv1d.py]: I was referring to uppercase D in the name - do we want to keep it uppercase?

Comment (pacospace) [reply tensorflow/conv1d.py]: It depends on the dimension of convolution you want to apply in general. There is also conv3d if we want. But I would keep it for later.

Comment (pacospace) [reply tensorflow/conv1d.py]: Updated with lower case, thanks @fridex :beers:

**Review by fridex — APPROVED** [review]
:beers:

**Pull request merged**